pkg/locale/localizedata/en.yaml

```diff
@@ -115,6 +115,7 @@ ConditionDeleted: "{{ .Field_1 }} condition has been deleted"
 Changed: "{{ .Field_1 }} has been changed"
 Updated: "{{ .Field_1 }} has been updated"
 DescriptionUpdated: "{{ .Field_1 }} description has been updated"
+EnvironmentRequireCommentUpdated: "Environment settings have been updated"
```

Comment:
How about adding `EnvironmentSettings` to [here](https://github.com/bucketeer-io/bucketeer/blob/ee1c0f51226730b0c8aa54067c37c339c9c2a95d/pkg/locale/localizer.go#L108) and using `locale.UpdatedTemplate`?

Reply:
@kentakozuka, done.